lib/facility_factory.rb

```diff
@@ -0,0 +1,39 @@
+class FacilityFactory
+
+  def create_or_facility(facility)
+    facility = facility.slice(:title,:location_1, :phone_number)
```

Comment:
We don't need to slice this up either. We can just adjust the keys we need, then just send the whole big hash into Facility.new since our initialize method in Facility is already set up to look for only the keys it needs.